lib/krane/deploy_task.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'kubernetes-deploy/deploy_task'
+
+module Krane
+  class DeployTask < KubernetesDeploy::DeployTask
```

Comment:
Does this implicitly get coverage from your CLI tests, or should we be using the krane version of the code a new integration test?

Reply:
It gets coverage form the black_box_tests